docs/how-to-guides/test-an-app.md

```diff
@@ -8,161 +8,168 @@ keywords: testing, qa, quality assurance, test runner
 
 # How to test an application
 
+!!! important
+    ksqlDB 0.30 ships with a new sql-based testing tool. For documentation on the old test runner,
+    switch to an older version of this page.
+
 ## Context
 
-You have an application of ksqlDB statements, and you want to automatically test whether they behave correctly when given a set of inputs. ksqlDB exposes a *test runner* command line tool to do just that. It runs quickly and doesn't require a running {{ site.aktm }} or ksqlDB cluster.
+ksqlDB ships with a command line tool to to test KSQL statements automatically.
+It runs quickly and doesn't require a running {{ site.aktm }} or ksqlDB cluster.
 
 ## In action
 
 ```bash
-ksql-test-runner -s statements.sql -i input.json -o output.json
+run-ksql-test --test-directory path/to/tests --temp-folder path/to/temp/folder
 ```
 
 ## Usage
 
-To test a set of SQL statements, you provide three files, one file
-containing the SQL statements and two JSON files containing the input
-records and the expected output records.
+To test a set of KSQL statements, provide a folder containing the sql test files and a folder
```

Comment:
s/and a folder/and another folder

(I'd also use 'directory' instead but it's just a personal preference for sure)